pom.xml

```diff
@@ -274,9 +274,9 @@
         <version.org.glassfish.javax.enterprise.concurrent>1.0</version.org.glassfish.javax.enterprise.concurrent>
         <version.org.glassfish.javax.json>1.1.2</version.org.glassfish.javax.json>
         <version.org.glassfish.soteria>1.0</version.org.glassfish.soteria>
-        <version.org.hibernate>5.3.2.Final</version.org.hibernate>
+        <version.org.hibernate>5.3.3.Final</version.org.hibernate>
         <version.org.hibernate.commons.annotations>5.0.4.Final</version.org.hibernate.commons.annotations>
-        <version.org.hibernate.search>5.10.1.Final</version.org.hibernate.search>        <version.org.hibernate.validator>6.0.10.Final</version.org.hibernate.validator>
+        <version.org.hibernate.search>5.10.3.Final</version.org.hibernate.search>        <version.org.hibernate.validator>6.0.10.Final</version.org.hibernate.validator>
```

Comment:
@yrodiere Please split those properties into two lines

Reply:
Seems like there was a pre-existing problem, the property for Hibernate Validator was actually duplicated on this line and the line below. I added a commit to fix that and force-pushed.